src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+const tableHead = table.querySelector('thead').querySelector('tr');
+const tableBody = table.querySelector('tbody').querySelectorAll('tr');
+const tableFoot = table.querySelector('tfoot').querySelector('tr');
+
+const th = document.createElement('th');
+
+th.innerText = tableHead.children[1].innerHTML;
+tableHead.children[4].before(th);
+
+const tf = document.createElement('th');
+
+tf.innerText = tableFoot.children[1].innerHTML;
+tableFoot.children[4].before(tf);
```

Comment:
there are code duplicates, so make sense to move duplicated code to the separate function and reuse it, for instance 
```suggestion
const tf = createElement('tf', tableFoot);
const th = createElement('th', tableHead);

function createElement(type, parent) {
 const element = document.createElement(type);

 element.innerText = parent.children[1].innerHTML;
 parent.children[4].before(tf);
}

```